src/content/blog/2023-09-20-tusd-200.md

```diff
@@ -0,0 +1,65 @@
+---
+title: 'tusd v2: More interoperable and resilient'
+author: acconut
+redirect_from: /blog/2023/09/20/tusd-200/
+date: 2023-09-20
+---
+Exactly fours years after the last major release of tusd, [version 1.0.0](https://tus.io/blog/2019/09/20/tusd-100), it is time for another big announcement: We are happy to say that tusd v2 is finished and ready for production!
```

Comment:
I would mention that this has been battle tested globally at scale at Transloadit.